mobile/src/main/java/rs/readahead/washington/mobile/media/MediaFileHandler.java

```diff
@@ -199,16 +200,24 @@ public static void exportMediaFile(Context context, VaultFile vaultFile, @Nullab
 
     public static Single<VaultFile> importPhotoUri(Context context, Uri uri, @Nullable String parentId) throws Exception {
         // Vault replacement methods
-        InputStream v_input = context.getContentResolver().openInputStream(uri); // original photo
-        Bitmap v_bm = modifyOrientation(BitmapFactory.decodeStream(v_input), v_input); // bitmap of photo
-        Bitmap v_thumb = ThumbnailUtils.extractThumbnail(v_bm, v_bm.getWidth() / 10, v_bm.getHeight() / 10); // bitmap of thumb
-
+        Boolean keepExif = Preferences.isKeepExif();
+        ByteArrayOutputStream v_image_jpeg_stream = new ByteArrayOutputStream();
         ByteArrayOutputStream v_thumb_jpeg_stream = new ByteArrayOutputStream();
```

Comment:
what do you think about renaming `v_image_jpeg_stream` to `imageJpegStream` and `v_thumb_jpeg_stream` to `thumbJpegStream` to follow Java naming conventions and improve readability?